src/cooldown.rs

```diff
@@ -122,6 +139,24 @@ impl CooldownTracker {
             self.member_invocations.insert((ctx.user_id, guild_id), now);
         }
     }
+    /// Sets the last invocation for the specified cooldown bucket.
```

Comment:
Maybe add a sentence that hints that you usually don't need this, that this is more of a special purpose sentence

`This function is usually not needed for regular usage. It was added for more flexibility in exceptional situations" or something like that. Since you made the PR, you probably even have some specific example use case :)

Reply:
Pushed a new commit that does this.